src/java.base/share/classes/java/util/Locale.java

```diff
@@ -70,131 +70,150 @@
 import sun.util.locale.provider.TimeZoneNameUtility;
 
 /**
- * A {@code Locale} object represents a specific geographical, political,
- * or cultural region. An operation that requires a {@code Locale} to perform
- * its task is called <em>locale-sensitive</em> and uses the {@code Locale}
- * to tailor information for the user. For example, displaying a number
- * is a locale-sensitive operation&mdash; the number should be formatted
- * according to the customs and conventions of the user's native country,
- * region, or culture.
- *
- * <p> The {@code Locale} class implements IETF BCP 47 which is composed of
+ * A {@code Locale} represents a specific geographical, political,
+ * or cultural region. An API that requires a {@code Locale} to perform
+ * its task is <em>locale-sensitive</em> and uses the {@code Locale}
+ * to tailor information for the user. These <em>locale-sensitive</em> APIs
+ * are principally in the <i>java.text</i> and <i>java.util</i> packages.
+ * For example, displaying a number is a <em>locale-sensitive</em> operation&mdash;
+ * the number should be formatted according to the customs and conventions of the
+ * user's native country, region, or culture.
+ *
+ * <p>The {@code Locale} class implements IETF BCP 47 which is composed of
  * <a href="https://tools.ietf.org/html/rfc4647">RFC 4647 "Matching of Language
  * Tags"</a> and <a href="https://tools.ietf.org/html/rfc5646">RFC 5646 "Tags
  * for Identifying Languages"</a> with support for the LDML (UTS#35, "Unicode
  * Locale Data Markup Language") BCP 47-compatible extensions for locale data
- * exchange.
- *
- * <p> A {@code Locale} object logically consists of the fields
- * described below.
+ * exchange. Each {@code Locale} is associated with locale data which is retrieved
+ * by the installed {@link java.util.spi.LocaleServiceProvider LocaleServiceProvider}
+ * implementations. Depending on the implementation, such data may vary by release.
+ *
+ * <h2 id="loc_comp">Locale Composition</h2>
+ * <p> A {@code Locale} is composed of the bolded fields described below; note that a
+ * {@code Locale} need not have all such fields. For example, {@link
+ * Locale#ENGLISH Locale.ENGLISH} is only comprised of the <em>language</em> field.
+ * In contrast, a {@code Locale} such as the one returned by {@code
+ * Locale.forLanguageTag("en-Latn-US-POSIX-u-nu-latn")} would be comprised of all
+ * the fields below. This particular {@code Locale} would represent English in
+ * the United States using the Latin alphabet and numerics for use in POSIX
```

Comment:
"script" is more precise than "alphabet", as the latter suggests only letters [A-Z]